dicekey/src/main/java/org/dicekeys/dicekey/DiceKey.kt

```diff
@@ -57,6 +63,7 @@ open class DiceKey<F: Face>(val faces: List<F>) {
       }
     }
 
+    // Note: not secure randomness is provided
```

Comment:
Perhaps:
> For testing only.  Not using a cryptographically secure random number generator.

(why not use a strong random number generator?)

Reply:
Fixed, use SecureRandom()